src/odemis/acq/stream/_live.py

```diff
@@ -1023,15 +1023,15 @@ def __init__(self, name, detector, dataflow, emitter, **kwargs):
         # Resolution assumes that we scan the whole roi. For smaller ROIs, the
         # actual hardware setting is proportional.
         self.resolution = model.TupleContinuous(hwres.value, range=hwres.range,
-                                                cls=(int, long, float),
+                                                cls=(int, float),
```

Comment:
Actually, `float` doesn't make sense for this VA, so just changing to `cls=int` would be even better.